testing/ef_tests/src/cases/bls_batch_verify.rs

```diff
@@ -0,0 +1,72 @@
+use super::*;
+use crate::case_result::compare_result;
+use crate::impl_bls_load_case;
+use bls::{verify_signature_sets, BlsWrappedSignature, PublicKeyBytes, Signature, SignatureSet};
+use serde_derive::Deserialize;
+use std::borrow::Cow;
```

Comment:
```suggestion
use std::borrow::Cow;
use std::str::FromStr;
```

Required for a later suggestion.